<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DelegatedColorMapSettingEditWidget</class>
 <widget class="QWidget" name="DelegatedColorMapSettingEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="useOriginalCheckBox">
     <property name="text">
      <string>Use Setting of Pre-processing Window</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidgetContainer" name="colorMapWidget" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="importButton">
       <property name="text">
        <string>&amp;Import...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportButton">
       <property name="text">
        <string>&amp;Export...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWidgetContainer</class>
   <extends>QWidget</extends>
   <header>misc/qwidgetcontainer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>useOriginalCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>importButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>52</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useOriginalCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>exportButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>433</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
